src/main/java/uk/gov/laa/ccms/caab/mapper/ApplicationMapper.java

```diff
@@ -486,7 +487,7 @@ default void finaliseLandAward(@MappingTarget LandAward landAward) {
   @AfterMapping
   default void finaliseOtherAssetAward(@MappingTarget OtherAssetAward otherAssetAward) {
     final TimeRecovery timeRecovery = otherAssetAward.getTimeRecovery();
-    otherAssetAward.setRecoveryOfAwardTimeRelated(String.valueOf(timeRecovery != null));
+    //TODO otherAssetAward.setRecoveryOfAwardTimeRelated(String.valueOf(timeRecovery != null));
```

Comment:
Was this not updated quite a while ago? `recoveryOfAwardTimeRelated` is now a boolean, so the `String.valueOf` is removed.

Reply:
fixed, was a merge issue somewhere